### PR TITLE
Adding the new version of the website with the teams page modified

### PR DESCRIPTION
**Pull request by jey12345678**

### CODE REVIEW

File: src/data/team/Finance-Team/jill-mezenberg.yaml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+id: jill-mezenberg
+first_name: Jill
+last_name: Mezenberg
+photo: images/Finance-Team-Pictures/jill-mezenberg.jpg
+photo_alt: Jill Mezenberg
+linkedin: "https://www.linkedin.com/in/fletcher-trinier-56288215b/"
```
Comment (Suri111200) [inline src/data/team/Finance-Team/jill-mezenberg.yaml:6]: Please replace with Jill's LinkedIn-- this is Fletcher's.

**Review by Suri111200 — CHANGES_REQUESTED** [review]
Please fix Jill's LinkedIn.

File: package.json [hunk 2]
```diff
@@ -16,7 +16,7 @@
     "chokidar": "^3.4.0",
     "classnames": "^2.2.6",
     "fs-extra": "^7.0.1",
-    "gatsby": "2.22.17",
+    "gatsby": "^2.22.17",
```
Comment (Suri111200) [inline package.json:19]: This change is also not desired and may break the site.

**Review by Suri111200 — CHANGES_REQUESTED** [review]
Please revert package.json

**Comment by Suri111200** [comment]
Also @jey12345678 please compress Michael's photo. It's 8.9 MB which is unnecessarily large

**Review by Suri111200 — APPROVED** [review]
LGTM. @AvaneeshM please review as well.

**Review by Suri111200 — CHANGES_REQUESTED** [review]
@AvaneeshM @jey12345678 please investigate failing tests-- not sure why that's the case. We can talk about this later if you would like

**Review by AvaneeshM — APPROVED** [review]

**Pull request merged**